Nodejs/Product/TestAdapterImpl/TestFrameworks/Hash.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+
+namespace Microsoft.NodejsTools.TestAdapter.TestFrameworks
+{
+    internal static class Hash
```

Comment:
Why implement our own hashing algorithm rather than use one in the .NET Framework (e.g. https://msdn.microsoft.com/en-us/library/xa627k19(v=vs.110).aspx ).

Reply:
This is a much faster algorithm, since it has none of the crypto-requirements. (And widely used by Roslyn, and the TypeScript LS).

Reply:
If SHA1/256 is fast enough for Git to check if an entire repo has any changes, then I'm sure its quick enough to go over a few test files. That said, never mind... it's not a major issue. (And I'd rather the the relative path anyway... which will be way faster again :wink: )